scripts/BuildAndTest.ps1

```diff
@@ -52,10 +52,16 @@ function Exit-WithFailureMessage($scriptName, $message) {
 }
 
 if (-not $NoBuild) {
+    Write-Information "Reformatting code..."
+    dotnet format --verify-no-changes $RepoRoot\src\Microsoft.Security.Utilities.sln --verbosity diagnostic
```

Comment:
Can we do this in PR validation, but not in BuildAndTest with no opt-in args? In my experience dotnet format is slow.

Reply:
Let`s leave this for now. We want to stop the formatting bleeding that keeps cropping up on every PR and then when we have that, we can tweak more leisurely. :)

Reply:
OK. I honor your deep care factor and we will follow your lead unless/until you run up against one of my own bugaboos. Like, C++ style parameter indentation. ;)